### PR TITLE
Add copy DTD URI action

### PR DESCRIPTION
**Pull request by helin24**
Fixes https://github.com/flutter/flutter-intellij/issues/7242

### CODE REVIEW

**Review by jwren — COMMENTED** [review]
LGTM

File: Dart/src/com/jetbrains/lang/dart/ide/toolingDaemon/DartToolingDaemonService.kt [hunk 1]
```diff
@@ -84,6 +85,7 @@ class DartToolingDaemonService private constructor(private val project: Project)
   private fun onServiceStarted(uri: String?, secret: String?) {
     serviceRunning = true
     this.secret = secret
+    this.uri = uri
```
Comment (alexander-doroshko) [inline Dart/src/com/jetbrains/lang/dart/ide/toolingDaemon/DartToolingDaemonService.kt:88]: Should we set `uri` back to `null` in `processTerminated()`, where `serviceRunning` becomes `false`?

File: Dart/src/com/jetbrains/lang/dart/ide/toolingDaemon/DartToolingDaemonService.kt [hunk 2]
```diff
@@ -48,6 +48,7 @@ class DartToolingDaemonService private constructor(private val project: Project)
   private lateinit var webSocket: WebSocket
   var webSocketReady = false
     private set
+  var uri: String? = null
```
Comment (alexander-doroshko) [inline Dart/src/com/jetbrains/lang/dart/ide/toolingDaemon/DartToolingDaemonService.kt:51]: I'll add `private set` to make sure that the value of this property can't be accidentally changed from elsewhere

**Comment by alexander-doroshko** [comment]
merged

File: Dart/src/com/jetbrains/lang/dart/ide/toolingDaemon/DartToolingDaemonService.kt (hunk 1 above)
Comment (helin24) [reply Dart/src/com/jetbrains/lang/dart/ide/toolingDaemon/DartToolingDaemonService.kt:88]: Ah yes, that would make sense. Same for in `onClose`?

**Comment by alexander-doroshko** [comment]
https://github.com/JetBrains/intellij-plugins/commit/3c0a035e4c2abfb416249edd434be0ea3446b58a

**Comment by helin24** [comment]
@alexander-doroshko do you know what version of the Dart plugin this will end up being in or when it will be released?

**Pull request closed without merging**